vnext/Desktop/module.g.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
```

Comment:
>pch [](start = 10, length = 3)

What generates this .g file?

Reply:
It is a "copy&paste" of file generated by C++/WinRT. For some reason, in this case it generates a file that has non-C++ identifiers and I had manually update it and make a copy.